packages/dashboard/src/DashboardService.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { Grid } from '@openmsupply-client/common';
+import { StockWidget } from './widgets';
 import {
-  InboundShipmentsWidget,
-  OutboundShipmentsWidget,
-  StockWidget,
-} from './widgets';
+  InboundShipmentWidget,
+  OutboundShipmentWidget,
+} from '@openmsupply-client/invoices';
```

Comment:
Maybe `lerna add @openmsupply-client/invoices --scope @openmsupply-client/dashbaord` to be eplicit?

Reply:
great! thanks